### ISSUE TITLE
Wyzecam3 not working

### ISSUE THREAD

**Issue by neatgit** [mrlt8/docker-wyze-bridge#1152]
I have a Wyze cam 3 and it is not working. I did not actually see the exact model number on the supported page of the wiki. IT's model number is WYZEC3. All it shows in bridge is "preview not available"

**Comment by mrlt8** [comment]
Is your bridge on the same network/LAN as the camera? There seems to be an issue with the latest version of the firmware blocking remote access to the v3 cams.

**Comment by HenryLai-Bayview** [comment]
Yes, I can confirm I am facing this issue as well. 

One WyzeCam v3 on 4.36.11.8391 and another WyzeCam v3 on 4.36.11.7095 that both show "The media could not be loaded, either because the server or network failed or because the format is not supported."

However, my WyzeCam v2 running firmware 4.9.9.1851 is working fine without issues.

**Comment by neatgit** [comment]
> Is your bridge on the same network/LAN as the camera? There seems to be an issue with the latest version of the firmware blocking remote access to the v3 cams.

Yes they are. Perhaps because I am using a mobile hot spot as my internet? I think mobile hot spot operates as some kind of proxy, which in turn does some strange things on networks.

Can the docker stream be viewed remotely on a safari browser? MY work phone doesn't allow me to download apps and I'd be watching the stream remotely.

**Comment by neatgit** [comment]
Do you first have to connect the camera to your wifi with the App? The whole reason I got this bridge is because my phone doesn't support the app

**Comment by mrlt8** [comment]
The bridge only looks for cameras already linked to a wyze account. You'll need to use the app to generate a QR code to activate and connect them to wifi.

**Comment by neatgit** [comment]
> The bridge only looks for cameras already linked to a wyze account. You'll need to use the app to generate a QR code to activate and connect them to wifi.

OK so I probably just need to install a virtual container to run android apps on linux correct? Have you tried this and succeeded? I cannot get the app on the phone. Any other hacks or workarounds?
